### DISCUSSION TITLE
Inserting pictures issues

### DISCUSSION THREAD

**Discussion by Nick222** [zim-desktop-wiki/zim-desktop-wiki#1810] [Ideas]
Can Zim optionally - when inserting picture:

1) Create white or another background under picture (if user change text background to dark-blue - some pictures very hard to look)

2) Create thin frame around picture

**Comment by quesada** [comment]
copy pasting a part of a note that contains a picture doesn't move the pic to the attachment folder of that note. So it's basically not a good workflow to work with zim and pictures, unless you know this and plan for it.

**Comment by Nick222** [comment]
> copy pasting a part of a note that contains a picture doesn't move the pic to the attachment folder of that note. So it's basically not a good workflow to work with zim and pictures, unless you know this and plan for it.

Can You create Your own issue - for such different problem?

**Comment by Nick222** [comment]
Move to Discussion, pls...

**Reply by introt** [reply]
You're welcome!

**Comment by quesada** [comment]
Seems to be solved in the current version, I just did this and it worked (0.74.2), Great job guys!

**Comment by treborxy** [comment]
@Nick222 : Very good point! Should be not too difficult, since css-similar tagging is used with other elements even now. For me it would be great if one could choose a default-style for images, e.g. 'max-width: 800px; border: 1px solid black;'
